pkg/filters/builder/responsebuilder_test.go

```diff
@@ -43,7 +43,7 @@ func getResponseBuilder(spec *ResponseBuilderSpec) *ResponseBuilder {
 	return rb
 }
 
-func setRequest(t *testing.T, ctx *context.Context, ns string, req *http.Request) {
+func setRequestWithNS(t *testing.T, ctx *context.Context, ns string, req *http.Request) {
```

Comment:
renaming to `setRequestWithNS` is not so elegant, we can reserve only one `func setRequest(t *testing.T, ctx *context.Context, ns string, req *http.Request)` to support `ns` parameter with nil value